internal/server/proto/server.proto

```diff
@@ -320,6 +320,8 @@ message Variable {
 
   oneof value {
```

Comment:
Docs on all of these would be good, particularly the `hcl` one :)

Reply:
Oooh documenting this pointed me to a bug. This will be a minute.

Reply:
OK, that was an embarrassing miss but yay for code reviews 🎉